packages/analytics/lib/modular/index.js

```diff
@@ -3,7 +3,7 @@ import { firebase } from '..';
 /**
  * Returns a Analytics instance for the given app.
  * @param app - FirebaseApp. Optional.
- * @returns {Analytics}
+ * @returns {import('..').FirebaseAnalyticsTypes.Module}
```

Comment:
Is this true? This does not seem like a valid change

Reply:
`Analytics` is not recognised. The linter gives the following error: `Cannot find name 'Analytics'.`. In the `d.ts` files, `Analytics` is an alias of `FirebaseAnalyticsTypes.Module`. Hence why I added this. The types work as expected

Reply:
resolved with typedefs per Russell's suggestion